app/services/platform-io.ts

```diff
@@ -292,4 +292,7 @@ export default class PlatformIO {
 
   static selectDirectoryDialog = (): Promise<any> =>
     nativeAPI.selectDirectoryDialog();
+
+  static shareFiles = (files: Array<string>): void =>
```

Comment:
I would like to call the shareFiles only on Cordova here, this way you do not have to change all other platform APIs.

Reply:
ok